### PR TITLE
TreeGrid scrolls to selected item on selection change

### PR DESCRIPTION
**Pull request by mpisacic**
Added scroll to support on TreeGrid and QuickGrid so that QuickGrid can be scrolled to certain position. On top of that scrolling to was implemented on TreeGrid so that when selected item on tree  is changed tree automatically scroll to some position so that selected item will be visible.

### CODE REVIEW

File: src/components/QuickGrid/QuickGrid.scss [hunk 1]
```diff
@@ -170,7 +170,8 @@
             }
 
             &.is-selected {
-                background-color: $disabled-field;
+                background-color: $selected-item-background;
+                font-weight: bold;
```
Comment (hbagarich) [inline src/components/QuickGrid/QuickGrid.scss]: Why are we changing style?

Comment (mpisacic) [reply src/components/QuickGrid/QuickGrid.scss]: so that selected item is better visible

File: src/components/TreeGrid/TreeGrid.tsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+import { ITreeGridProps, ITreeGridState } from './TreeGrid.Props';
+import { getTreeRowsSelector } from './treeGridDataSelectors';
 
 export class TreeGrid extends React.PureComponent<ITreeGridProps, ITreeGridState> {
     public static defaultProps = {
         isNodeSelectable: true
     };
 
-    private _quickGrid: any;
+    private readonly _rowOverselect = 0;
```
Comment (vinkobedek) [inline src/components/TreeGrid/TreeGrid.tsx]: what does this do if it is always zero?

Comment (mpisacic) [reply src/components/TreeGrid/TreeGrid.tsx]: nothing much, can be removed

**Pull request merged**